modules/UI/videolayout/Filmstrip.js

```diff
@@ -444,7 +444,7 @@ const Filmstrip = {
 
             promises.push(new Promise(() => {
                 let { localThumb } = this.getThumbs();
-                let height = localThumb.height();
+                let height = localThumb ? localThumb.height() : 0;
```

Comment:
No noticeable display issues occur from passing 0 and forcing small icons? I'm guessing it could look a little off in horizontal filmstrip mode, assuming the filmstrip is displayed, but I'm guessing not enough to address.

Reply:
You're probably right. The exception I was fixing appears when the filmstrip is hidden in iAmRecorder mode, where the local thumbnail is undefined. Not sure if the local thumbnail could ever be undefined in other circumstances...

Reply:
This is going away soon, so I may just add a comment here what we're trying to fix. WDYT?

Reply:
If it's going away soon then I don't think it's necessary to add a comment.